test/test_with_xpress.jl

```diff
@@ -32,6 +32,170 @@ using Random
 Random.seed!(42)  # for test consistency, random prices used in FlexibleHVAC tests
 
 
+@testset "Emissions" begin
```

Comment:
@hdunham the tests are taking over 5 hours! Can we reduce these tests?

Reply:
That's strange, they didn't take long on my machine. Will look into it.

Reply:
Could the long solve times be caused due to REopt version? I tested "Thermal loads" testset and got the following solve times:

- in Emissions branch: 56 seconds for first try, then 15 seconds for each attempt.
- in Develop branch: 45 seconds for first try, then 9 seconds for each attempt afterwards

would it makes sense to merge develop into emissions?

Reply:
The testset solve times should be fixed per https://github.com/NREL/REopt.jl/commit/b3b3d490c1eea19a3e6611f4287e34f579a57c5e. along with some cleanup of error conditions. Downstream code adds RE electricity constraints if min re elec pct is nothing. But we were defaulting min re elec pct to 0.0 so it was never nothing, and hence always adding min re elec constraints.